sql/sql_class.h

```diff
@@ -71,8 +71,8 @@ void set_thd_stage_info(void *thd,
 
 #include "wsrep.h"
 #include "wsrep_on.h"
-#ifdef WITH_WSREP
 #include <inttypes.h>
+#ifdef WITH_WSREP
```

Comment:
(At least) 3 of the Buildbots doesn’t understand `PRIuPTR` (now replaced with `PRiPTR`) for
https://github.com/MariaDB/server/pull/3360#discussion_r1703517880.

Let me know if I should `#include <inttypes.h>` somewhere else.